packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2401,12 +2401,13 @@ export class ContainerRuntime
 	 * @returns - routes of objects deleted from the runtime
 	 */
 	public deleteUnusedNodes(unusedRoutes: string[]): string[] {
```

Comment:
naming: "unused" can have different meanings, including unreferenced in the moment.
I think you want to use "collectable" kind of adjective, and give it a definition somewhere (routes that have been unreferenced for 30 days)

Reply:
We can call it sweepReadyRoutes or deleteReadyRoutes. ContainerRuntime should not be concerned with the criteria for makes a route sweep ready. We can add a comment in the Garbage Collector class that says what makes routes sweep ready.

Reply:
Renamed to deleteSweepReadyNodes